src/include/RequestHandler.h

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) 2021 Pei-Ru Wang, Cheryl Huang, Yuan Zhou. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+/**
+ * Project MyHttpServer
+ * @author Pei-Ru Wang, Cheryl Huang, Yuan Zhou
+ * @version 0.0.1
+ */
+
+#ifndef _REQUESTHANDLER_H
+#define _REQUESTHANDLER_H
+
+#include "HttpHandler.h"
+#include "HttpRequestPacket.h"
+#include "HttpResponsePacket.h"
+
+// Basic RequestHandler, father of business handlers
+class RequestHandler {
+public:
+  RequestHandler();
```

Comment:
Maybe set this to default so you don't have to implement it.
`RequestHandler() = default;`